client/app/components/compose.coffee

```diff
@@ -311,9 +312,13 @@ module.exports = Compose = React.createClass
 
         # new draft
         else
-            state = messageUtils.makeReplyMessage @props.selectedAccountLogin,
-                @props.inReplyTo, @props.action,
-                @props.settings.get('composeInHTML')
+            state = messageUtils.makeReplyMessage(
+                @props.selectedAccountLogin,
+                @props.inReplyTo,
+                @props.action,
+                @props.settings.get('composeInHTML'),
+                @props.selectedAccount.get('signature')
```

Comment:
I think it's best to pass the selectedAccount object and retrieve fields from it.